Source_code/dev_gpio/include/dev_gpio.h

```diff
@@ -0,0 +1,13 @@
+
+#ifndef dev_gpio_h
+#define dev_gpio_h
+
+// define IOC commands 
+#define DEV_GPIO_IOC_MAGIC  'K'
+#define DEV_GPIO_IOC_RESET    _IO(GPIO_DEV_IOC_MAGIC, 0)
+
+
+
+
+
+#endif
```

Comment:
May want to make this 
`#endif /* DEV_GPIO_H */`

just to make it a bit clear what this statement is ending